source/moon-resolution.js

```diff
@@ -69,6 +71,11 @@
 			type = type || screenType;
 			if (type) {
 				enyo.dom.addBodyClass('moon-res-' + type.toLowerCase());
+				enyo.dom.addBodyClass('enyo-res-' + type.toLowerCase());
```

Comment:
`moon-res-*` should remain in 2.5-upkeep because app code may already be using it, so we can't take it away. We _can_ add the future name of the class, so apps which are able to update, can update now to protect them from future changes.